src/ripple/app/paths/Pathfinder.cpp

```diff
@@ -92,6 +92,23 @@ bool comparePathRank (
     return a.index > b.index;
 }
 
+// Compare two PathRanks based on liquidity.
+// A better PathRank is lower, so the best are sorted to the beginning.
+bool comparePathLiquidity(
+    Pathfinder::PathRank const& a, Pathfinder::PathRank const& b)
```

Comment:
This is so close to a duplicate of `comparePathRank`, why not have `comparePathRank` call `comparePathLiquidity`?